docs/pages/database-access/guides/rds.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Database Access with AWS RDS and Aurora for PostgreSQL and MySQL
-description: How to configure Teleport Database Access with AWS RDS and Aurora for PostgreSQL and MySQL.
+title: Database Access with AWS RDS and Aurora for PostgreSQL, MySQL and MariaDB
```

Comment:
To prevent this title from being truncated in Google's search results, we should get this to around 55 chars or less, e.g., by making it "Database Access with AWS RDS and Aurora." We can then add an `h1` attribute to the frontmatter that uses the original title, "Database Access with AWS RDS and Aurora for PostgreSQL, MySQL and MariaDB." The `h1` attribute is not used to populate the page's `title` element, while `title` is.

Reply:
Good tip. I'd never think about it. Changed.